app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,173 @@
 // Code your JavaScript / jQuery solution here
+var turn = 0;
+var topRow;
+var middleRow;
+var bottomRow;
+var firstColumn;
+var secondColumn;
+var thirdColumn;
+var botToTopDiagonal;
+var topToBottomDiagonal;
+var completeBoard;
+var saveButton;
+var previousGamesButton;
+var clearButton;
+
+function player () {
```

Comment:
Consider using an arrow function along with a ternary for this function 
`const player = () => turn % 2 === 0 ? 'X' : 'O'`